src/realm/array_backlink.cpp

```diff
@@ -181,7 +183,7 @@ ObjKey ArrayBacklink::get_backlink(size_t ndx, size_t index) const
         return ObjKey(int64_t(value >> 1));
     }
 
-    Array backlink_list(m_alloc);
+    BPlusTree<int64_t> backlink_list(m_alloc);
     backlink_list.init_from_ref(ref_type(value));
```

Comment:
Is this going to interpret the data correctly if it is a simple Array that hasn't been split yet?